### ISSUE TITLE
clear up journal

### ISSUE THREAD

**Issue by parmentelat** [parmentelat/nbhosting#37]
in production, it seems like the logs as exposed through `journalctl --unit=nbh-uwsgi`

is littered with lines of the form

```
Nov 10 17:28:21 thurst sudo[20799]:     root : TTY=unknown ; PWD=/root/nbhosting/nbhosting ; USER=0a503ef365ff5853e5e59641da52ae3e ; COMMAND=/bin/mkdir -p /nbhosting/students/0a503ef365ff5853e5e59641da52ae3e/flotpython3
Nov 10 17:28:21 thurst sudo[20807]:     root : TTY=unknown ; PWD=/root/nbhosting/nbhosting ; USER=0a503ef365ff5853e5e59641da52ae3e ; COMMAND=/bin/mkdir -p /nbhosting/students/0a503ef365ff5853e5e59641da52ae3e/flotpython3
Nov 10 17:28:21 thurst sudo[20818]:     root : TTY=unknown ; PWD=/root/nbhosting/nbhosting ; USER=0a503ef365ff5853e5e59641da52ae3e ; COMMAND=/bin/mkdir -p /nbhosting/students/0a503ef365ff5853e5e59641da52ae3e/flotpython3
Nov 10 17:28:21 thurst sudo[20836]:     root : TTY=unknown ; PWD=/root/nbhosting/nbhosting ; USER=0a503ef365ff5853e5e59641da52ae3e ; COMMAND=/bin/mkdir -p /nbhosting/students/0a503ef365ff5853e5e59641da52ae3e/flotpython3
```

generally it is 4 or 5 lines of that kind each time an event occurs, it would help a lot to keep the system from emitting these messages that don't bring any valuable information

**Comment by parmentelat** [comment]
along the same line, it would make sense to clean up the outputs of the shell script that is too verbose at this point

**Comment by parmentelat** [comment]
update on this issue:

* first kind of spurrious output (mkdir -p) was fixed in 459cfed5344eb025dca225ea036f14b2ca5ecaa6

**Comment by parmentelat** [comment]
good enough for now
